spec/models/parser_spec.rb

```diff
@@ -121,30 +121,69 @@
   end
 
   describe "#enrichment_definitions" do
+    let(:version) { FactoryGirl.build(:version) }
     let(:parser_class) { mock(:parser_class, enrichment_definitions: {ndha_rights: "Hi"} )}
     let(:loader) { mock(:loader, loaded?: true, parser_class: parser_class).as_null_object }
 
     before(:each) do
       parser.stub(:loader) { loader }
     end
 
-    it "returns the parser enrichment definitions" do
-      loader.parser_class.enrichment_definitions("staging").should eq({ndha_rights: "Hi"})
-    end
+    context 'when version is not passed' do
+      it "content is the last content which is set in the Parser itself" do
```

Comment:
Maybe subject { FactoryGirl.build(:parser) } or implicit using Parser, rather than
let(:parser) { FactoryGirl.build(:parser) }; Then you could use "is_expected" without specifying the parser every time. Nonetheless, not a serious issue.